include/jemalloc/internal/sz.h

```diff
@@ -61,7 +61,7 @@ sz_psz2ind(size_t psz) {
 		pszind_t lg_delta = (x < LG_SIZE_CLASS_GROUP + LG_PAGE + 1) ?
 		    LG_PAGE : x - LG_SIZE_CLASS_GROUP - 1;
 
-		size_t delta_inverse_mask = ZD(-1) << lg_delta;
+		size_t delta_inverse_mask = ZD(ZU(-1) << lg_delta);
```

Comment:
I think we don't want a ZD at all, right? No sense in casting an unsigned to a signed back to an unsigned.

Reply:
Yeah this doesn't really do anything. But I think we could get warnings from -Wsign-conversion w/o it?

Reply:
Will we? It's unsigned shifted by unsigned, right? Or a ZU(lg_delta).